config.yml

```diff
@@ -0,0 +1,178 @@
+teams:
+- name: fwos-maintainers
+  maintainers:
+  - tkuhrt
+  members:
+  - jorgefl0
+  - skullquake
+- name: identity-credential-admins
+  maintainers:
+  - davidz25
+  members: []
+- name: identity-credential-maintainers
+  maintainers:
+  - davidz25
+  members:
+  - 15characterlimi
+  - dritan-x
+  - dyiop
+  - GarethCOliver
+  - sethmoo
+  - sorotokin
+  - suzannajiwani
+- name: multiformat-vc-ios-maintainers
+  maintainers:
+  - tkuhrt
+  members:
+  - bhavyachauhan-ping
+  - gauravping
+- name: sd-jwt-dotnet-maintainers
+  maintainers:
+  - thomas-tran
+  members: []
+- name: sd-jwt-js-admin
+  maintainers: []
+  members:
+  - cre8
+  - lukasjhan
+- name: sd-jwt-js-maintainers
+  maintainers:
+  - lukasjhan
+  - tkuhrt
+  members:
+  - aceshim
+  - berendsliedrecht
+  - cre8
+  - zustkeeper
+- name: sd-jwt-kotlin-maintainers
+  maintainers:
+  - fabian-hk
+  - tkuhrt
+  members: []
+- name: sd-jwt-python-maintainers
+  maintainers:
+  - danielfett
+  - tkuhrt
+  members:
+  - peppelinux
+- name: sd-jwt-rust-maintainers
+  maintainers:
+  - tkuhrt
+  members:
+  - jovfer
+- name: vc-api-developers
+  maintainers: []
+  members:
+  - jrhender
+  - nichonien
+- name: vc-api-maintainers
```

Comment:
Does this need a `members: []` line added?

Reply:
nope